### DISCUSSION TITLE
Cloning from command working, but from app is not

### DISCUSSION THREAD

**Discussion by leandrodcarvalho** [community/community#122742] [Repositories]
### Select Topic Area

Bug

### Body

I'm not able to use the app for a repository.
When I try to clone it, I got this:

`Cloning into 'C:\test'...
remote: Enumerating objects: 799096, done.        
remote: Counting objects: 100% (1356/1356), done.        
remote: Compressing objects: 100% (926/926), done.        
error: RPC failed; curl 56 Failure when receiving data from the peer
error: 4610 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output`

It works normally from command line, and since it worked I was trying to just add an existing repo to the app.
But then when I try to commit something for example, I get this:

`test-git-hooks//pre-commit: line 9: $'\r': command not found
test-git-hooks//pre-commit: line 10: $'\r': command not found
test-git-hooks//pre-commit: line 13: $'\r': command not found
test-git-hooks//pre-commit: line 24: syntax error: unexpected end of file`

Also working fine from command line for commit.

Any help?

**Comment by ziroziro** [comment]
I have the same issue, anyone know to solve?
